admin.py

```diff
@@ -107,6 +109,38 @@ class TicketInline(admin.TabularInline):
     extra = 0
 
 
+def process_names_csv(csv_file):
+    """
+    Process a csv file for online orders
+    @param csv_file: A csv file containing names to be set to paid.
+    @return: list of names that had problems being processed and a message summarizing the done process.
+    """
+    names = pandas.read_csv(csv_file, delimiter="\t", header=None)
+    open_tickets = OnlineOrder.objects.filter(performance__production__active=True).all()
+
+    error_msgs = []
+    amount_processed_success = 0
+    amount_processed_fail = 0
+    for index, data in names.iterrows():
+        first_name = data[0]
+        last_name = data[1]
+        person_tickets = open_tickets.filter(first_name__iexact=first_name, last_name__iexact=last_name)
+        if not person_tickets.exists():
+            error_msgs.append(f"'{first_name} {last_name}' was not found to have tickets in active performances.")
+            amount_processed_fail += 1
+            continue
+        person_tickets = person_tickets.all()
+
+        for ticket in person_tickets:
```

Comment:
I don't think this is a clever idea since I notice a lot of double orders by accident!